cli/src/commands/simplify_parents.rs

```diff
@@ -0,0 +1,119 @@
+use std::collections::HashSet;
+
+use itertools::Itertools;
+use jj_lib::backend::BackendResult;
+use jj_lib::revset::RevsetExpression;
+use jj_lib::rewrite::CommitRewriter;
+use jj_lib::settings::UserSettings;
+
+use crate::cli_util::CommandHelper;
+use crate::cli_util::RevisionArg;
+use crate::command_error::CommandError;
+use crate::ui::Ui;
+
+/// Simplify parent edges for the specified revision(s).
+///
+/// Removes all redundant parent edges going into any of the specified
```

Comment:
(Optional) The following seems a bit clearer to me for the first sentence:

> Removes any parents of each of the the specified revisions that are also indirect ancestors of the same revision

For the second sentence, I'd also be more decisive:

> This has no effect on any revision's contents or on the working copy.

I'm also tempted to write "This should have no effect on any...", and I'm OK with that if you prefer it, but I think style guides or Orwell's "Politics and the English language" would suggest dropping the "should"

Reply:
Rewored this mostly according to the suggestions.